### PR TITLE
Fix keystone ldap check for LP 1896125

### PR DESCRIPTION
**Pull request by dosaboy**
python3-keystone is installed as a dep for all openstack services so we need to check for package "keystone" to know this is a keystone host and we also now check that ldap is actually in use.

### CODE REVIEW

File: defs/scenarios/openstack/keystone/bugs.yaml [hunk 1]
```diff
@@ -11,6 +11,14 @@ checks:
         # Victoria
         - min: 2:18.0.0
           max: 2:18.0.0-99
+    config:
+      handler: hotsos.core.plugins.openstack.OpenstackConfig
+      path: etc/keystone/keystone.conf
```
Comment (mustafakemalgilor) [inline defs/scenarios/openstack/keystone/bugs.yaml:16]: The [ldap] option can be turned on per-domain basis [1], so if we want this to work properly we also need to check all .conf files under `/etc/keystone/domains/` as well.

[1] See `Additional LDAP integration settings` section at https://docs.openstack.org/keystone/zed/admin/configuration.html#identity-ldap-server-set-up

Comment (mustafakemalgilor) [reply defs/scenarios/openstack/keystone/bugs.yaml:16]: The domain names can be arbitrary so I'm unsure if doing this is possible right now, we would need to allow wildcards in the path.

Comment (dosaboy) [reply defs/scenarios/openstack/keystone/bugs.yaml]: Good catch! I have now added support to do this and updated the patch.

Comment (dosaboy) [reply defs/scenarios/openstack/keystone/bugs.yaml]: ugh just realised my domain config path is not valid, ill fix that now

Comment (dosaboy) [reply defs/scenarios/openstack/keystone/bugs.yaml]: done.

**Review by mustafakemalgilor — APPROVED** [review]

**Pull request merged**